tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -927,6 +944,30 @@ export class WebGPUBackend extends KernelBackend {
     return this.tensorMap.numDataIds() - this.tensorDataPendingDisposal.length;
   }
 
+  /**
+   * Returns shader generated by func.
+   * @param inputSpecs Required parameters for operators.
+   * @param func Optional, call operators according to input spec.
+   */
+  // tslint:disable-next-line: no-any
+  getShader(inputSpecs: any, func?: (inputSpecs: any) => void):
+      {[key: string]: string} {
+    env().set('WEBGPU_CPU_FORWARD', false);
```

Comment:
```   
 const savedFlag = tf.env().getBool('WEBGPU_CPU_FORWARD');
 tf.env().set('WEBGPU_CPU_FORWARD', true);
```
And at the end the this func, call ` tf.env().set('WEBGPU_CPU_FORWARD', sevedFlag);`